src/widgets/QDisassemblyView.cpp

```diff
@@ -1287,7 +1287,7 @@ void QDisassemblyView::paintEvent(QPaintEvent *) {
 		});
 
 		// find suitable arrow horizontal width
-		for (int i = 0; i < jump_arrow_vec.size(); i++) {
+		for (int i = 0; i < (int) jump_arrow_vec.size(); i++) {
```

Comment:
Isn't `std::vector<>` indexed by an unsigned type like `size_t` or `std::vector<>::size_type`? It's hard to tell, but if `i` is never compared to a signed value, then we can probably just make it a `size_t` and not need this cast.

Also, even though it's more verbose, please prefer c++ style casts :-).

Reply:
Ah yes, you're correct. Fixed.